android/build.gradle

```diff
@@ -38,6 +38,5 @@ def supportVersion = rootProject.hasProperty('supportLibVersion') ? rootProject.
 
 dependencies {
     compile 'com.facebook.react:react-native:+'
-    compile "com.android.support:customtabs:${supportVersion}"
+    compile "com.android.support:customtabs:26.0.1"
```

Comment:
Do the variables above that reference `rootProject` check the properties in our apps? project?  If so I'm wondering if we can work around this by updating our SDK version to the minimum one needed for the required chrome customtabs support and then set that version in our project's `build.gradle` using something like `rootProject.setProperty('supportLibVersion', '26.0.1')` so that this would reference our specified version without needing to modify the file?

Either way we'll likely want to update the SDK version across the board to the minimum version needed for proper support.